src/main/java/GeneralService/Controller.java

```diff
@@ -0,0 +1,17 @@
+package GeneralService;
+
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+@RestController
+public class Controller {
+    public ArrayList<String> validCarUserIds = new ArrayList<>(Arrays.asList("50075", "15080"));
```

Comment:
Always use List as the object type to keep it more generic, even when you know the list will always be an arrayList, always make it List type.
Also remove the instantiation of the object for now, we will populate this List from the file of users